examples/models/transform_jitter.py

```diff
@@ -1,3 +1,18 @@
+"""
+==========================================
+Jittered Scatter Plot with Bokeh and NumPy
+==========================================
+
+This example demonstrates how to add jitter to a scatter plot using Bokeh and NumPy.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.models.sources.ColumnDataSource, bokeh.models.markers.Circle, bokeh.models.Jitter, bokeh.models.LabelSet, bokeh.models.widgets.Button, bokeh.models.callbacks.CustomJS
+    :refs: :ref:`userguide_plotting` > :ref:`userguide_plotting` > :ref:`userguide_widgets`, :ref:`userguide_styling` > :ref:`userguide_annotations`
```

Comment:
These references need to match the exact references defined in the RST files for the docs. For example, in basic.rst, the label is ``ug_basic``, so this is what should go here (see the example in https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations).
These references here are also supposed to work like breadcrumbs, i.e. first the general section (like `ug_interaction`) and then the specific subsection (like `ug_interaction_widgets`

```suggestion
    :refs: :ref:`ug_interaction` > :ref:`ug_interaction_widgets`, :ref:`ug_topics` > :ref:`ug_topics_categorical_scatters_jitter`
```